plan/planningDocument.md

```diff
@@ -0,0 +1,95 @@
+# Site Plan - dms.com
+
+## Overview
+
+*dms.com* is a website for the business 'Diversified Manufacturing and Services' (DMS). The goal of this site is to showcase the company, detail the company's products and services, and provide methods of contacting the business. The site will be a professional platform to increase the amount of requests to the business for their specific products and services.
+
+This site will target prospective clients that are looking for an electronics manufacturer. Today it is expected that a legitimate business have a professional-looking website to inform the client and bridge their communication. Therefore, it is necessary that DMS provide an online presence to provide information about the business and facilitate client communication.
+
+The site will be primarily static and built from scratch, custom to the professional environment DMS wishes to project. The content will span several pages and cover a general description of the company, specific manufacturing and design services, and methods of contacting and requesting quotes. The website will focus on responsive design and have a fluid, professional look on all platforms. The site will complement the logo and images for the company which will consist of pc boards and industrial equipment by providing cool green tones and light and dark pastel-like brown colors. Some such colors are #effcef, #ccedd2, #94d3ac, #655c56, and #eeecda 
```

Comment:
When I mentioned the colors for the site, I intended for you to put this material in a "design" section in a document along with your wireframes and site map. It's okay, just thought I would mention it so you can include it there as well.